utils/polus-python-template/{{cookiecutter.container_name}}/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[tool.poetry]
+name = "{{cookiecutter.project_name}}"
+version = "{{cookiecutter.plugin_version}}"
+description = "{{cookiecutter.plugin_description}}"
+authors = ["{{cookiecutter.author}} <{{cookiecutter.author_email}}>"]
+readme = "README.md"
+packages = [{include = "polus", from = "src"}]
+
+[tool.poetry.dependencies]
+python = "^3.9.15"
+typer = "^0.7.0"
+bfio = {version = "2.1.9", extras = ["all"]}
+filepattern = "2.0.0"
+preadator = "0.3.0.dev1"
+
+[[tool.poetry.source]]
+name = "test"
+url = "https://test.pypi.org/simple/"
+default = false
+secondary = true
```

Comment:
This should be removed. We added this when `filepattern2` was only available from `testpypi`